test/lambda/api/index.test.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayProxyEventV2, Context, Callback } from "aws-lambda";
+import { indexHandler } from  '../../../lib/lambda/api/index'
+
+describe('Unit test for app handler', function () {
+    it('Verifies 404 response for invalid route', async () => {
+        const event: APIGatewayProxyEventV2 = {
+            rawPath: '/',
+            queryStringParameters: {
+                a: "1"
+            },
+            requestContext: {
+                http: {
+                method: "POST",
+            }
+        }
```

Comment:
```suggestion
        const event: APIGatewayProxyEventV2 = {
            rawPath: '/',
            queryStringParameters: {
                a: "1"
            },
            requestContext: {
                  http: {
                      method: "POST",
                  }
            }
```
Your indentation is off.